.github/workflows/versionControlAndAuditCheck.yml

```diff
@@ -393,10 +393,27 @@ jobs:
           echo "This step will make sure that an audit is logged for each contract modified/added by this PR."
           echo "It will also make sure that no information is missing in the audit log and that the information is meaningful."
 
-          # load list of protected contracts
-          PROTECTED_CONTRACTS=$(cat contracts_for_audit.txt)
+          echo "cat contracts_for_audit.txt:"
+          echo cat contracts_for_audit.txt
+          echo ""
+
+          # convert from comma-separated to newline-separated
+          CONTRACTS_FOR_AUDIT_STR=$(tr ',' '\n' < contracts_for_audit.txt)
+          echo "CONTRACTS_FOR_AUDIT_STR: $CONTRACTS_FOR_AUDIT_STR"
+          PROTECTED_CONTRACTS="$CONTRACTS_FOR_AUDIT_STR"
```

Comment:
if you want to keep the PROTECTED_CONTRACTS variable name due to later implementation in the script, why not call it PROTECTED_CONTRACTS directly, instead of creating the CONTRACTS_FOR_AUDIT_STR var?